src/execution/sql/ddl_executors.cpp

```diff
@@ -106,6 +106,12 @@ bool DDLExecutors::CreateIndexExecutor(const common::ManagedPointer<planner::Cre
                      *(node->GetSchema()));
 }
 
+bool DDLExecutors::CreateIndexExecutor(const common::ManagedPointer<const planner::CreateIndexPlanNode> node,
```

Comment:
This looks identical to the above code. Can you un-const it where it is being called? If not, can you remove the other version?

Reply:
removed